enterprise-suite/es-grafana/kubernetes-exporter.json

```diff
@@ -1,7 +1,38 @@
 [
-   {"graphName": "CPU Percent", "promQL":["node_cpu_percent"]},
-   {"graphName": "Memory Available Percent", "promQL":["node_memory_usable_percent"]},
-   {"graphName": "File System Free Percent", "promQL":["node_filesystem_free_percent"]},
-   {"graphName": "Bytes Received Per Second", "promQL":["sum by (instance) (irate(node_network_receive_bytes[1m]))"]},
-   {"graphName": "Bytes Transmitted Per Second", "promQL":["sum by (instance) (irate(node_network_transmit_bytes[1m]))"]}
+    {
+        "graphName": "CPU Percent",
+        "promQL": [
+            "sum without(container_name,id,image,name)((irate(container_cpu_user_seconds_total{ContextTags}[1m]) + irate(container_cpu_system_seconds_total[1m])) * 100)"
+        ]
+    },
+    {
+        "graphName": "Active Memory Used Percent",
+        "promQL": [
+            "sum without(container_name,id,image,name)((container_memory_working_set_bytes{ContextTags} / container_spec_memory_limit_bytes < +Inf or container_memory_working_set_bytes{ContextTags} / on (node_ip) group_left node_memory_MemTotal) * 100)"
+        ]
+    },
+    {
+        "graphName": "Total Memory Used Percent",
+        "promQL": [
+            "sum without(container_name,id,image,name)((container_memory_usage_bytes{ContextTags} / container_spec_memory_limit_bytes < +Inf or container_memory_usage_bytes{ContextTags} / on (node_ip) group_left node_memory_MemTotal) * 100)"
+        ]
+    },
+    {
+        "graphName": "File System Used Percent",
+        "promQL": [
+            "sum without(container_name, id, image, name)(container_fs_usage_bytes{ContextTags} / container_fs_limit_bytes * 100)"
```

Comment:
Isn't this going to produce a metric for every filesystem on the node? I seem to remember having to filter this down somehow.

Reply:
The cadvisor collection seems to be pretty clean and sticking to real devices used in the container.

node_exporter definitely has a bunch of cruft to sweep through but we do it this way

https://github.com/lightbend/helm-charts/blob/master/enterprise-suite/templates/node-exporter.yaml#L42-L43